### PR TITLE
Makefiles: Include clean in help message

### PR DESCRIPTION
**Pull request by Blendify**
Subject: The make clean help command was missing from all make files.

### CODE REVIEW

File: sphinx/templates/quickstart/Makefile_t [hunk 1]
```diff
@@ -94,4 +95,3 @@ gettext:
 .PHONY: Makefile
 %: Makefile
 	$(SPHINXBUILD) -b "$@" $(ALLSPHINXOPTS) "$(BUILDDIR)/$@"
-
```
Comment (tk0miya) [inline sphinx/templates/quickstart/Makefile_t]: Why removing this?

**Review by tk0miya — COMMENTED** [review]
LGTM with nits.

File: sphinx/templates/quickstart/Makefile_t (hunk 1 above)
Comment (Blendify) [reply sphinx/templates/quickstart/Makefile_t]: I believe my editor was configured to strip more that 1 trailing white line at the end of the file. The file had two blank lines at the bottom. If this change is too noisy I can revert it.

Comment (tk0miya) [reply sphinx/templates/quickstart/Makefile_t]: It seems our test case expects a carriage return before EOF. Please revert this line or update a test case.

**Comment by tk0miya** [comment]
Thank you for the update. Merging now.

**Pull request merged**